Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_V1_0.h

```diff
@@ -23,7 +23,7 @@
 
 #include "pins_BTT_SKR_MINI_E3.h"
 
-#define BOARD_INFO_NAME "BIGTREE SKR Mini E3"
+#define BOARD_INFO_NAME "BTT SKR Mini E3"
```

Comment:
For consistency with the other E3 Mini boards, I changed this to "BTT SKR Mini E3 V1.0".